prism/ext/lpsolve55/Makefile

```diff
@@ -24,26 +24,26 @@ endif
 
 default: all
 
-all: checks ../../lib/$(LIBPREFIX)lpsolve55$(LIBSUFFIX)
+all: checks ../../$(PRISM_LIB_DIR)/$(LIBPREFIX)lpsolve55$(LIBSUFFIX)
 
 # Try and prevent accidental makes (i.e. called manually, not from top-level Makefile)
 checks:
 	@if [ "$(LIBSUFFIX)" = "" ]; then \
 	  (echo "Error: This Makefile is designed to be called from the main PRISM Makefile"; exit 1) \
 	fi; 
 
-../../lib/$(LIBPREFIX)lpsolve55$(LIBSUFFIX):
+../../$(PRISM_LIB_DIR)/$(LIBPREFIX)lpsolve55$(LIBSUFFIX):
 	mkdir -p lib
 	@(if [ "$(OSTYPE)" = "cygwin" ]; then \
-	  cp bin/$(BINDISTSUFFIX)/$(LIBPREFIX)lpsolve55$(LIBSUFFIX) ../../lib; \
+	  cp bin/$(BINDISTSUFFIX)/$(LIBPREFIX)lpsolve55$(LIBSUFFIX) ../../$(PRISM_LIB_DIR); \
 	else \
-	  (cd src/lp_solve_5.5/lpsolve55 && \
-	  ARCH=$(ARCH) c=$(C) sh -x $(LPSOLVE55_BUILD) \
-	  ) && mv lib/$(LIBPREFIX)lpsolve55$(LIBSUFFIX) ../../lib/; \
+	  (cd $(PRISM_SRC_DIR)/lp_solve_5.5/lpsolve55 && \
```

Comment:
`PRISM_SRC_DIR` should stay as `src` here: it refers to the local directory

Reply:
Well-spotted - fixed in 584fb28.